SeaPublicWebsite/Controllers/EnergyEfficiencyController.cs

```diff
@@ -57,8 +57,11 @@ public IActionResult NewOrReturningUser_Post(NewOrReturningUserViewModel viewMod
                     ModelState.AddModelError(nameof(NewOrReturningUserViewModel.Reference), "Check you have typed the reference correctly. Reference must be 8 characters.");
                     return View("NewOrReturningUser", viewModel);
                 }
+
+                var userData = userDataStore.LoadUserData(viewModel.Reference);
+                var recommendations = RecommendationService.GetRecommendationsForUser(userData);
                 
-                return RedirectToAction("YourSavedRecommendations_Get", "EnergyEfficiency", new { reference = viewModel.Reference });
+                return RedirectToAction("Recommendation_Get", "EnergyEfficiency", new { reference = viewModel.Reference, id = (int) recommendations[0].Key });

```

Comment:
What if the user doesn't have any recommendations?
I've left the same question on the issue.

Reply:
This is what it was originally there :) From how the code was originally written, it is guaranteed to show a hardcoded suggestion, but the BRE integration doesn't enforce this anymore...